### DISCUSSION TITLE
No way to see Fief version from admin dashboard

### DISCUSSION THREAD

**Discussion by fubuloubu** [fief-dev/fief#385] [Ideas]
## Describe the bug

I couldn't find a way to figure out the version of the Fief software being used from within the dashboard

## To Reproduce

1. Go to '/admin'
2. No Fief version displayed

## Expected behavior

Displaying the version somewhere (maybe About tab or something)

## Configuration

* Cloud or self-hosted: self-hosted (Railway)
* If self-hosted, Fief version: 0.28.6

## Additional context

I made another issue and had to guess at what version I was running based on the ghcr `latest` tag

**Comment by frankie567** [comment]
Why not :) For the record, the version is announced in the server logs at startup!

**Reply by fubuloubu** [reply]
Noticed this after posting!
